test/gtest/ucp/test_ucp_peer_failure.cc

```diff
@@ -134,6 +145,21 @@ void test_ucp_peer_failure::init() {
         smoke_test();
     }
     wrap_errors();
+
+    ucp_ep_params_t ep_params_mod = {0};
+    ep_params_mod.field_mask = UCP_EP_PARAM_FIELD_ERR_HANDLER_CB |
+                               UCP_EP_PARAM_FIELD_USER_DATA;
```

Comment:
Where do we test the error flow ? (attempt to update a field that it is cost)?

Reply:
Do you mean this (https://github.com/openucx/ucx/pull/2130/files/7ac6bc63bf7d2e8a9775ef8578ed6833e3d783af#diff-4fb58bb922ffb07d1d1682f0129c0aa4R179)?